functions/global-teardown.ts

```diff
@@ -1,5 +1,6 @@
 import { teardown } from 'jest-dev-server'
 
 module.exports = async function globalTeardown() {
+  await new Promise((resolve) => setTimeout(resolve, 5000))
```

Comment:
Why is this timeout necessary? Shouldn't all of the tests be done by this point?

Reply:
From what I've deduced, the 2 remaining sources of flakiness are:
1. the server just starting improperly which I haven't been able to figure out why this happens (only happens 1/10 runs)
2. The socket closing before tests finish.

I was troubleshooting the 2nd cause and thought that might help it, but I'm pretty sure it was because of jest competing with the proxy. Will remove and test!